public/style.css

```diff
@@ -121,33 +121,177 @@ main {
     justify-content: center;
-    padding: 1em 0;
+    align-items: center;
 }
 
-header div {
-    max-width: 1200px;
+.navbar-menu li{
+    font-size: 3em;
+    line-height: auto;
+    text-transform: none;
 }
 
-header img {
-    width: 10em;
-    height: auto;
+.navbar-menu a{
+    color: var(--primary-color-1);
 }
 
-header ul,
-header input {
-    display: none;
+.navbar-donate {
+    color: var(--active-color-1);
+    font-weight: 900;
 }
 
+#navbar-toggle .navbar-menu-button {
+  display: block;
+  width: 33px;
+  height: 4px;
+  margin-bottom: 5px;
+  position: relative;
+  
+  background: var(--primary-color-1);
+  border-radius: 3px;
+  
+  z-index: 1;
+  
+  transform-origin: 4px 0px;
+  
+  transition: 
+  transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0), 
+  background 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+  opacity 0.55s ease;
+}
 
+#navbar-toggle .navbar-menu-button:first-child {
+  transform-origin: 0% 0%;
+}
 
+#navbar-toggle .navbar-menu-button:nth-last-child(2) {
+  transform-origin: 0% 100%;
+}
+
+#navbar-toggle input:checked ~ .navbar-menu-button {
+  opacity: 1;
+  transform: rotate(45deg) translate(-2px, -1px);
+}
+
+#navbar-toggle input:checked ~ .navbar-menu-button:nth-last-child(3) {
+  opacity: 0;
+  transform: rotate(0deg) scale(0.2, 0.2);
+}
+
+#navbar-toggle input:checked ~ .navbar-menu-button:nth-last-child(2) {
+  transform: rotate(-45deg) translate(0, -1px);
+}
+
+#navbar-toggle input{ 
+    display: block;
+    width: 40px;
+    height: 32px;
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    opacity: 0;
+    z-index: 2;
+}
+
+#navbar-toggle input:checked ~ .navbar-menu {
+    right: 0;
+    transform: none;
+}
 
 
+@media (min-width: 768px) {
+
+    header {
+        border-bottom: none;
+    }
+
+    header div {
+        flex-direction: column;
+        max-width: 1200px;
+        gap: 1em;
+    }
+
+    header div nav{
+        width: 100%;
+    }
+
+    .navbar-menu {
+        background: none;
+        max-width: unset;
+        border-top: var(--border-style-black);
+        border-bottom:var(--border-style-black);
+        padding: .5em 1em;
+
+    }
+
+    .navbar-menu li {
+        font-size: .7em;
+        letter-spacing: 1px;
+        text-transform: uppercase;
+    }
+ 
+
+    .navbar-menu {
+        position:unset;
+        top: 0;
+        right: 0;
+        width: unset;
+        height: unset;
+        background: none;
+        transition: none;
+        overflow-y: unset;
+    
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    #navbar-toggle .navbar-menu-button,
+    #navbar-toggle input {
+        display: none;
+    }
+
+    #navbar-toggle input:checked ~ .navbar-menu-button,
+    #navbar-toggle input:checked ~ .navbar-menu-button:nth-last-child(3),
+    #navbar-toggle input:checked ~ .navbar-menu-button:nth-last-child(2) {
+        transform: unset;
+    }
+}
+
+@media (min-width: 1024px) {
```

Comment:
Kan je hier ook een container query voor gebruiken? Is dat beter, fijner, handiger?